ui/src/frontend/time_selection_panel.ts

```diff
@@ -154,6 +153,7 @@ export class TimeSelectionPanel extends Panel {
       }
     }
 
+    ctx.fillStyle='blue';
```

Comment:
I haven't managed to see how these primary colours are used in hovering on the time selection panel, but it's odd that they are hard-coded where the general pattern of this PR is themify with variables. Should these colours not also be abstracted by variables in the theme?

Reply:
I was trying to test something with that, but I also didn't see a change and must have forgot to remove them